runtime/include/localeModels/knl

```diff
@@ -0,0 +1 @@
+numa
```

Comment:
The symbolic link is consistent with runtime/include/cygwin{32,64} which I thought was a good way to show that this part of the code doesn't change for KNL -- i.e. file hierarchy as documentation.